client/lib/abtest/active-tests.js

```diff
@@ -61,4 +61,12 @@ module.exports = {
 		},
 		defaultVariation: 'notOffered'
 	},
+	monthlyPlanPricing: {
+		datestamp: '20160118',
```

Comment:
You might want to set a date in the past to ease testing since the A/B test won't run for accounts created prior to this date. You can check how @drewblaisdell did that in https://github.com/Automattic/wp-calypso/pull/2453.